packages/extension/src/utils/processAndStoreDocumentCookies.ts

```diff
@@ -53,6 +53,8 @@ const processAndStoreDocumentCookies = async ({
           parsedCookie.domain = singleCookie.domain?.startsWith('.')
             ? singleCookie.domain
             : '.' + singleCookie.domain;
+        } else {
+          parsedCookie.domain = window.location.hostname;
```

Comment:
It is not immediately obvious to me that how does this change help in this PR's aim. Would you mind adding some context in the PR description?

Reply:
It was a small bug fix that was noticed while self-testing the PR so added this here.